CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+## Questions / Bugs / Feedback
+
+If you have questions about the library, found a bug or want to suggest a feature, please create an issue.
+
+## Development
+
+- Install dependencies:
+
+```bash
+npm install
```

Comment:
Should this be `npm ci`?

Reply:
Changed!

> The vercel build link at the bottom of the PR is broken for me

It doesn't work because the docs don't exist in this PR yet. 😅  If you check #2 it's all good.

Reply:
Yep - realized that after the fact 😂